SparseGrids/tasgridExternalTests.cpp

```diff
@@ -1357,7 +1357,39 @@ bool ExternalTester::testAllRefinement() const{
     }
     cout << "      Construction              dynamic/local" << setw(15) << ((pass4) ? "Pass" : "FAIL") << endl;
 
-    return (pass && pass2 && pass3 && pass4);
+    bool pass5 = true;
+    {
+        const BaseFunction *f = &f21c1c2periodic;
```

Comment:
If the function is C^1 - C^2 then how come the anisotropy is (3, 4) as opposed to (2, 3).

Reply:
Functions in C^k (i.e., continuous and periodic derivatives up to order k) have Fourier coefficients c_n that decay like O(n^-(k+2)). So we should have alpha=(1+2, 2+2)=(3, 4).